<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>providerSourcerStampConverter</class>
 <widget class="QMainWindow" name="providerSourcerStampConverter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Provider Sourcing/Stamp Converter</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="0">
     <widget class="QLabel" name="hostnameLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hostname (v2/v3):</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="nameLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Provider Name (v1):</string>
      </property>
      <property name="buddy">
       <cstring>nameEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Sources:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="pathLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Path (v2):</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="12">
     <widget class="QLineEdit" name="ipportEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="13">
     <widget class="QLineEdit" name="sourceEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="10">
     <widget class="QPushButton" name="addSourceButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/plus.png</normaloff>:/images/plus.png</iconset>
      </property>
     </widget>
    </item>
    <item row="2" column="2" colspan="11">
     <widget class="QLineEdit" name="stampEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="11">
     <widget class="QComboBox" name="protocolVersionBox">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <item>
       <property name="text">
        <string>v0 | Plain DNS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v1 | DNSCrypt</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v2 | DNS over HTTPS</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>v3 | DNS over TLS</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="11">
     <widget class="QPushButton" name="removeSourceButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="images.qrc">
        <normaloff>:/images/minus.png</normaloff>:/images/minus.png</iconset>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>IP/Port:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="9">
     <widget class="QCheckBox" name="doesntFilter">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Doesn't filter</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLabel" name="hashesLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Hashes (v2/v3 comma seperated):</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="9">
     <widget class="QCheckBox" name="supportsDNSSEC">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Supports DNSSEC</string>
      </property>
     </widget>
    </item>
    <item row="1" column="12">
     <widget class="QPushButton" name="updateSourceButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Update</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Stamp:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="pubkeyLabel">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Provider Pub Key (v1): </string>
      </property>
      <property name="buddy">
       <cstring>pubkeyEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="10" column="1" colspan="12">
     <widget class="QLineEdit" name="nameEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="14" column="0" colspan="13">
     <widget class="QTreeWidget" name="autosourcesTree">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
      <property name="uniformRowHeights">
       <bool>true</bool>
      </property>
      <attribute name="headerDefaultSectionSize">
       <number>200</number>
      </attribute>
      <column>
       <property name="text">
        <string>Listed Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Stamp</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="5" column="0" colspan="9">
     <widget class="QCheckBox" name="doesntLog">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Doesn't log</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Protocol Version:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1" colspan="12">
     <widget class="QLineEdit" name="pubkeyEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="12" column="2" colspan="11">
     <widget class="QLineEdit" name="hostnameEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="11" column="2" colspan="11">
     <widget class="QLineEdit" name="hashesEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="13" column="2" colspan="11">
     <widget class="QLineEdit" name="pathEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="8">
     <widget class="QComboBox" name="sourcesBox">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="addItButton">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ADD IT!</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sourceEdit</tabstop>
  <tabstop>addSourceButton</tabstop>
  <tabstop>removeSourceButton</tabstop>
  <tabstop>updateSourceButton</tabstop>
  <tabstop>sourcesBox</tabstop>
  <tabstop>stampEdit</tabstop>
  <tabstop>protocolVersionBox</tabstop>
  <tabstop>doesntFilter</tabstop>
  <tabstop>doesntLog</tabstop>
  <tabstop>supportsDNSSEC</tabstop>
  <tabstop>ipportEdit</tabstop>
  <tabstop>pubkeyEdit</tabstop>
  <tabstop>nameEdit</tabstop>
  <tabstop>hashesEdit</tabstop>
  <tabstop>hostnameEdit</tabstop>
  <tabstop>pathEdit</tabstop>
  <tabstop>autosourcesTree</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
